dz8/operations.py

```diff
@@ -0,0 +1,88 @@
+def exit_program():
+    print('\nЗавершение программы')
+    exit()
+
+
+def yes_no():
+    print('\n1) Да\n'
+          '2) Нет\n')
+    option = int(input('Введите ваш вариант: '))
+    while option > 2 or option < 1:
+        option = int(input('Выберете ваш вариант: '))
+    return option
+
+
+def creating_base():
+    data = str(input("\nВведите имя файла базы данных: "))
```

Comment:
```suggestion
    data = input("\nВведите имя файла базы данных: ")
```
Функция ввод данных всегда возвращает объект типа `str`